.ci/docker/common/install_conda.sh

```diff
@@ -109,7 +109,8 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   # gcc-9 for ubuntu-18.04 from http://ppa.launchpad.net/ubuntu-toolchain-r/test/ubuntu
   # Pulls llibstdc++6 13.1.0-8ubuntu1~18.04 which is too new for conda
   # So remove libstdc++6.so.3.29 installed by https://anaconda.org/anaconda/libstdcxx-ng/files?version=11.2.0
-  if grep 18.04.6 /etc/issue >/dev/null; then
+  # Same is true for gcc-12 from Ubuntu-22.04
+  if grep -e [12][82].04.[62] /etc/issue >/dev/null; then
```

Comment:
lol, this is wicked

Reply:
It matches 18.04.6 and 22.04.2, and everything in between :)